File: AndroidApps/AndroidPortfolid/app/src/main/java/com/example/androidportfolid/MainActivity.java
package com.example.androidportfolid;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.ListView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        RecyclerView list = findViewById(R.id.recycler_view_profile);

        // Project list
        Project[] projects = {
                new Project("Start", "Very First", R.drawable.getting_started),
                new Project("Vin", "Very First", R.drawable.quote),
                new Project("BMI", "Very First", R.drawable.calculator),
                new Project("Inches", "Very First", R.drawable.tape),
                new Project("Angry Hungry", "Very First", R.drawable.hungry_developer)
        };

        ProfileAdapter adapter = new ProfileAdapter(projects);
        list.setAdapter(adapter);
    }
}

File: AndroidApps/ENamecard/app/src/main/java/com/jinco/enamecard/MainActivity.kt
package com.jinco.enamecard

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ArrayAdapter
import com.jinco.enamecard.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)
        setSpinner()
    }

    private fun setSpinner() {
        val jopSpinner = arrayOf("one", "two")
        val jopAdapter = ArrayAdapter(this, android.R.layout.simple_spinner_dropdown_item, jopSpinner)
        binding.spJob.adapter = jopAdapter
    }
}